src/Skoruba.IdentityServer4.Admin/Helpers/DbMigrationHelpers.cs

```diff
@@ -31,12 +34,51 @@ public static void EnsureSeedData(IServiceProvider serviceProvider)
             using (var scope = serviceProvider.GetRequiredService<IServiceScopeFactory>().CreateScope())
             {
                 var context = scope.ServiceProvider.GetRequiredService<AdminDbContext>();
+                var userManager = scope.ServiceProvider.GetRequiredService<UserManager<UserIdentity>>();
+                var roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<UserIdentityRole>>();
+
                 context.Database.Migrate();
-                EnsureSeedData(context);
+                EnsureSeedIdentityServerData(context);
+                EnsureSeedIdentityData(userManager, roleManager);
+            }
+        }
+
+        /// <summary>
+        /// Generate default admin user / role
+        /// </summary>
+        private static void EnsureSeedIdentityData(UserManager<UserIdentity> userManager,
+            RoleManager<UserIdentityRole> roleManager)
+        {
+            // Create admin role
+            if (!roleManager.RoleExistsAsync(AuthorizationConsts.AdministrationRole).Result)
+            {
+                var role = new UserIdentityRole { Name = AuthorizationConsts.AdministrationRole };
+
+                roleManager.CreateAsync(role).Wait();
```

Comment:
Why is this method not `async`? Used in a non-awaitable context?

Reply:
Great point! 
I am wondering about it as well. This method is executed in `Main` method from `Program.cs`. What do you recommend for this use case - try to rewrite it using async/await? I'm not sure how it will work. I've found similiar question on SO - https://stackoverflow.com/a/9212343/943403 - what do you think about it?

Reply:
See the very last code block in that answer; `async Main` should work with the latest .NET Core.

You may need to have to set the language level to `Latest`:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#edit-the-csproj-file

Reply:
It works! :) 
Please take a look here - https://github.com/skoruba/IdentityServer4.Admin/blob/master/src/Skoruba.IdentityServer4.Admin/Program.cs#L10-L17 - this async method called `DbMigrationHelpers.EnsureSeedData(host)` is commented by default and if I add async keyword in `Main` then I get a warning about lacks of await keyword. Is there some potential side effect?

Reply:
It should be fine, just change `public static void Main` to `public static async Task Main` and then you can `await` in it.